recipe/activate-clang_win-64.sh

```diff
@@ -102,25 +102,26 @@ else
   LIB_USED="${CONDA_PREFIX}/lib"
 fi
 
+CHOST=@CHOST_BASE@@CL_VERSION@
```

Comment:
This line prevents backing up CHOST correctly in _tc_activation. You can just do
```
sed -i.bak "s|@CHOST@|${CHOST_BASE}${CL_VERSION}}|g" ${CHANGE}-${PKG_NAME}.sh
```
in install-pkg.sh